frontend/src/FullPageRoutes/TokenPage.tsx

```diff
@@ -31,7 +31,7 @@ const useStyles = makeStyles({
 export default function TokenPage(): JSX.Element {
   const [displayedForm, setDisplayedForm] = React.useState<string | null>(null)
   const [logged_in, setLoggedIn] = React.useState<boolean>(false)
-  const [email, setEmail] = React.useState<string | null>(null)
+  const [, setEmail] = React.useState<string | null>(null)
```

Comment:
Is this a typo? Where is the first value?
```suggestion
  const [email, setEmail] = React.useState<string | null>(null)
```

Reply:
This is not a typo. `email` is not used, so it doesn't need to be assigned.

Reply:
Okay so if email is never used, then why is there a state for it? Will it be used later but right now, it is not being used?

Reply:
There are definitely things email will be used for. But we probably haven't gotten to them yet.

Reply:
> 
> 
> Okay so if email is never used, then why is there a state for it? Will it be used later but right now, it is not being used?

That's a good point. However, more fun to come that will make this obsolete

Reply:
Other PRs will fix this